recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,357 @@
+  {% with subpackage_id = "fasttext" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      ## NOTE: fasttext in NOT available for windows on conda-forge
+      skip: True  # [win]
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "fasttext": base_packages + fasttext_packages
+        - {{ name_ }} {{ version }}
+        - fasttext {{ libver_fasttext }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+  
+  ### spacy_packages
+  {% if use_spacy_packages is defined %}
+  {% if use_spacy_packages or use_all_subpackages %}
+  {% with subpackage_id = "spacy" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "spacy": base_packages + spacy_packages
+        - {{ name_ }} {{ version }}
+        - spacy {{ libver_spacy }}
+        - spacy-lookups-data {{ libver_spacy_lookups_data }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+  
+  # s2v_packages
+  {% if use_s2v_packages is defined %}
+  {% if use_s2v_packages or use_all_subpackages %}
+  {% with subpackage_id = "sense2vec" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # s2v_packages = ["sense2vec>=1.0.2"] + spacy_packages
+        # "sense2vec": base_packages + s2v_packages
+        - {{ pin_subpackage(name_ ~ "-spacy", exact=True) }}
+        - sense2vec {{ libver_sense2vec }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+
+  ### tf_packages
+  {% if use_tf_packages is defined %}
+  {% if use_tf_packages or use_all_subpackages %}
+  {% with subpackage_id = "tfhub" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+    # NOTE: this throws error for windows build.
+      skip: True  # [win]
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "tfhub": base_packages + tf_packages
+        - {{ name_ }} {{ version }}
+        - tensorflow {{ libver_tensorflow }}
+        - tensorflow-hub {{ libver_tensorflow_hub }}
+        # Adding the follwoing temporarily to fix a 
+        # tensorflow-dependency version-mismatch issue.
+        - absl-py {{ "~=0.10" }}
+        - wrapt {{ "~=1.12.1" }}
+      run_constrained:
+        # NOTE: tensorflow-text is not available on conda-forge
+        - tensorflow-text {{ libver_tensorflow_text }}
```

Comment:
Same as in https://github.com/conda-forge/staged-recipes/pull/16963#issuecomment-974666530

Reply:
Done.